TbsCore/Core/WebBrowserInfo.cs

```diff
@@ -185,18 +276,24 @@ public void Dispose()
                     Driver.Quit(); // Also disposes
                     Driver = default;
                 }
-                catch (Exception e)
+                catch (WebDriverException)
+                {
+                    Driver.Quit();
+                    Driver = default;
+                }
+                catch (Exception)
                 {
-                    // broswer closed because user or crash ??
-                    if (e.Message.Contains("chrome not reachable"))
-                    {
-                        Driver.Quit(); // Also disposes
-                        Driver = default;
-                    }
                 }
             }
 
             chromeService.Dispose();
         }
     }
+
+    // just for catching
+    // if this is anti-pattern or bad desgin
+    // tell me, i'm still learning ._.
+    public class NoChromeException : Exception

```

Comment:
not sure what you are trying to achieve here. And remove the comment.

Reply:
i want read - able exception ._.